certified-connectors/iManage Work/readme.md

```diff
@@ -0,0 +1,47 @@
+# iManage Work
```

Comment:
Hello @v-EgorKozhadei The template requires that we specify instructions on how to deploy the connector as a custom connector. We don't envisage iManage Work connector to be deployed as a custom connector, primarily because we use OAuth2, which would require us to expose the client-id and client-secret. Can we skip this part in the template?

Reply:
@v-EgorKozhadei Fixed